vernac/g_vernac.mlg

```diff
@@ -65,8 +65,7 @@ let parse_compat_version = let open Flags in function
   | "8.10" -> Current
   | "8.9" -> V8_9
   | "8.8" -> V8_8
-  | "8.7" -> V8_7
-  | ("8.6" | "8.5" | "8.4" | "8.3" | "8.2" | "8.1" | "8.0") as s ->
+  | ("8.7" | "8.6" | "8.5" | "8.4" | "8.3" | "8.2" | "8.1" | "8.0") as s ->
```

Comment:
This should be a catch-all.

Reply:
Why? Currently there is a difference between old versions not supported anymore and versions that were never supported. Furthermore, it seems that it is not the right time to request such a change so close to the freeze. This can also be done later on.

Reply:
> there is a difference between old versions not supported anymore and versions that were never supported

What's the point of that difference?

> it seems that it is not the right time to request such a change so close to the freeze

Why not?

Reply:
These changes are autogenerated by the compatibility script.  Making this change requires both changing the file and updating the script to recognize the new code and to not keep adding things here.  I don't have time to do that today, and I don't think this should delay merging this, and it is important to merge this before branching.  There will be another PR like this submitted shortly after the branch; feel free to make the request there and I'll update things.

Reply:
Ok, sounds good, this is not urgent obviously.